### DISCUSSION TITLE
updateLineItems in [shopify-buy] has incorrect input

### DISCUSSION THREAD

**Discussion by nrandell** [DefinitelyTyped/DefinitelyTyped#64959] [Issues with a @types package]
It should take an array of `CheckoutLineItemUpdateInput` but takes an array of `CheckoutLineItemInput`

This means calls fail as the id is a required input.

[a comment by an automated account is not shown]

**Comment by straiforos** [comment]
> It should take an array of `CheckoutLineItemUpdateInput` but takes an array of `CheckoutLineItemInput`
> 
> This means calls fail as the id is a required input.

Wow, great find. If you'd be willing to make a PR with updates I will review them. I worked on version 1.0, seems like they are up to 2.18.0.

Kind regards,

Stephen

**Comment by nrandell** [comment]
PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64974 created!
